sirepo/feature_config.py

```diff
@@ -174,7 +174,7 @@ def b(msg, dev=False):
             "codes that contain proprietary information and authorization to use is granted through oauth",
         ),
         raydata=dict(
-            file_reply_tmp_dir=pkconfig.RequiredUnlessDev(
+            file_reply_tmp_dir=(
```

Comment:
I do not understand why this is here.

Reply:
Where would be a better place? It is used abstractly inside of job_cmd so adding the configuration there doesn't make sense. Should we separate configuration for sim_types out of feature_config and into a module for each sim_type?